bpf/lib/eps.h

```diff
@@ -10,16 +10,22 @@
 #include "maps.h"
 
 static __always_inline __maybe_unused struct endpoint_info *
-lookup_ip6_endpoint(struct ipv6hdr *ip6)
+__lookup_ip6_endpoint(const union v6addr *ip6)
```

Comment:
Could we find another function name? The leading `__` tends to make me believe this function shouldn't be called from outside of `lookup_ip6_endpoint()`.

Maybe something along `lookup_ip6_endpoint_from_daddr()`?